swap/swap.go

```diff
@@ -155,6 +153,7 @@ func New(dbPath string, prvkey *ecdsa.PrivateKey, backendURL string, params *Par
 	if backendURL == "" {
 		return nil, errors.New("no backend URL given")
 	}
+	swapLog.SetLogAction("init")
 	swapLog.Info("connecting to SWAP API", "url", backendURL)
```

Comment:
as previously mentioned, `swaplog` should not be showing; the swap action should be showing instead.

the `base` is used in order to identify nodes in case a simulation logs stuff (see #1623)